tests/test_status.py

```diff
@@ -0,0 +1,112 @@
+import datetime
+import pytz
+
+from uuid import uuid4
+import pytest
+from django.test import TestCase
+from django.conf import settings
+
+from nyc.models import NYCBill
+from councilmatic_core.models import Action
+
+class TestStatus(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I'd prefer that our tests use a JSON file in coordination with a fixture that loads data ([for example](https://github.com/datamade/la-metro-councilmatic/blob/master/lametro/tests/conftest.py)). If we start writing more tests, it will be easier to have the data, the db setup, and the tests in discrete files. 

What do you think? any strong preferences?